test/common.bash

```diff
@@ -148,8 +153,8 @@ ch_version_docker=$(echo "$ch_version" | tr '~+' '--')
 # despite the admonition in the man page, because it's more portable [1].
 #
 # [1]: https://unix.stackexchange.com/a/136527
-ch_imgdir=$(readlink -ef "$CH_TEST_IMGDIR")
-ch_tardir=$(readlink -ef "$CH_TEST_TARDIR")
+ch_imgdir=$(readlink -ef "$CH_TEST_IMGDIR") || fatal "CH_TEST_IMGDIR does not exist"
+ch_tardir=$(readlink -ef "$CH_TEST_TARDIR") || fatal "CH_TEST_TARDIR does not exist"
```

Comment:
I think the thing to do here is make a wrapper function for `readlink(1)` that complains if the directory doesn't exist.

It should also report which directory does not exist, not just the variable containing the name of the nonexistent directory.